.github/CODEOWNERS

```
@@ -2,16 +2,98 @@
+/packages/permission-log-controller    @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers
+/packages/selected-network-controller  @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers @MetaMask/metamask-assets
+
+## Package Release related
+/packages/accounts-controller/package.json        @MetaMask/accounts-engineers @MetaMask/wallet-framework-engineers
+/packages/accounts-controller/CHANGELOG.md        @MetaMask/accounts-engineers @MetaMask/wallet-framework-engineers
+/packages/address-book-controller/package.json    @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/address-book-controller/CHANGELOG.md    @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/announcement-controller/package.json    @MetaMask/wallet-ux @MetaMask/wallet-framework-engineers
+/packages/announcement-controller/CHANGELOG.md    @MetaMask/wallet-ux @MetaMask/wallet-framework-engineers
+/packages/approval-controller/package.json        @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/approval-controller/CHANGELOG.md        @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/assets-controller/package.json          @MetaMask/metamask-assets @MetaMask/wallet-framework-engineers
+/packages/assets-controller/CHANGELOG.md          @MetaMask/metamask-assets @MetaMask/wallet-framework-engineers
+/packages/ens-controller/package.json             @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/ens-controller/CHANGELOG.md             @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/gas-fee-controller/package.json         @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/gas-fee-controller/CHANGELOG.md         @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/keyring-controller/package.json         @MetaMask/accounts-engineers @MetaMask/wallet-framework-engineers
+/packages/keyring-controller/CHANGELOG.md         @MetaMask/accounts-engineers @MetaMask/wallet-framework-engineers
+/packages/logging-controller/package.json         @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/logging-controller/CHANGELOG.md         @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/message-manager/package.json            @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/message-manager/CHANGELOG.md            @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/name-controller/package.json            @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/name-controller/CHANGELOG.md            @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/notification-controller/package.json    @MetaMask/notifications @MetaMask/wallet-framework-engineers
+/packages/notification-controller/CHANGELOG.md    @MetaMask/notifications @MetaMask/wallet-framework-engineers
```

```suggestion
/packages/notification-controller/package.json    @MetaMask/snaps-devs @MetaMask/wallet-framework-engineers
/packages/notification-controller/CHANGELOG.md    @MetaMask/snaps-devs @MetaMask/wallet-framework-engineers
```
Looks like this was missed in the recent iteration!